packages/account/src/Components/poi-unverified/__test__/unverified.spec.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { BrowserRouter } from 'react-router-dom';
+import { PlatformContext } from '@deriv/shared';
+import { Unverified } from '../unverified.jsx';
+
+jest.mock('@deriv/components', () => {
+    const original_module = jest.requireActual('@deriv/components');
+    return {
+        ...original_module,
+        Icon: jest.fn(() => <div data-testid='mockedIcon' />),
+    };
+});
+
+describe('Unverified', () => {
+    const renderWithRouter = component =>
+        render(
+            <PlatformContext.Provider value={{ is_appstore: true }}>
+                <BrowserRouter>{component}</BrowserRouter>
```

Comment:
We don't have any links in Unverified component. We can ignore BrowserRouter


render(
            <PlatformContext.Provider value={{ is_appstore: true }}>
               {component}
            </PlatformContext.Provider>
        );